### PR TITLE
Address severe concern regarding user-facing docs

### PR DESCRIPTION
**Pull request by md678685**
0/10 literally unplayable please pull immediately

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ WorldEdit is Minecraft mod that turns Minecraft into an in-game map editor (sort
 * Making fjords and other natural wonders with weird names
 * Also: fix grief, broken water, missing snow, raging fires, and other :poop:
 
-WorldEdit is open source!, so if you've just learned how to make a button blink, I probably recommend picking up a book on CS algorithms. The project is licensed using the GNU Lesser General Public License license, v3.
+WorldEdit is open source! So, if you've just learned how to make a button blink, I probably recommend picking up a book on CS algorithms. The project is licensed using the GNU Lesser General Public License license, v3.
```
Comment (wizjany) [inline README.md]:
```suggestion
**WorldEdit is open source**, so if you've just learned how to make a button blink, I probably recommend picking up a book on CS algorithms. The project is licensed using the GNU Lesser General Public License license, v3.
```

Comment (md678685) [reply README.md]: This removes the exclamation! It totally undermines the emotional impact of the sentence! I think it's way more important to preserve the exclamation even if it changes the semantics!

Comment (q220) [reply README.md]: I think I'm going to request a CVE because that's how severe this is. It threatens my grammatical security.

**Comment by md678685** [comment]
This project currently lacks an English linting tool. A quick search of NPM brings up [several potential packages](https://www.npmjs.com/search?q=english%20lint&ranking=quality) that we could use for unit testing the README.

File: README.md (hunk 1 above)
Comment (me4502) [reply README.md]: I agree with wiz's suggestion, it reads better.

**Pull request closed without merging**